Symfony/CS/Tests/Fixer/Contrib/Php4ConstructorFixerTest.php

```diff
@@ -71,11 +71,14 @@ public function Bar()
     }
 }
 
-class Foo
+namespace
```

Comment:
still no reason to change meaning of code, split it into two:

https://3v4l.org/8s3qk
https://3v4l.org/Esg9u